controllers/database/api/postgresql_type.go

```diff
@@ -0,0 +1,211 @@
+package api
+
+// Postgres CRD definition, please use CamelCase for field names.
+
+import (
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Postgresql defines PostgreSQL Custom Resource Definition Object.
+type Postgresql struct {
```

Comment:
Why create  `Postgresql` struct, does it copy from https://github.com/zalando/postgres-operator/blob/master/pkg/apis/acid.zalan.do/v1/postgresql_type.go
If true, please add some comments to explain it.